moveit_reach_plugins/src/evaluation/manipulability_moveit.cpp

```diff
@@ -141,9 +169,93 @@ double ManipulabilityRatio::calculateScore(const Eigen::MatrixXd& jacobian_singu
   return jacobian_singular_values.minCoeff() / jacobian_singular_values.maxCoeff();
 }
 
+bool ManipulabilityNormalized::initialize(XmlRpc::XmlRpcValue& config)
+{
+  bool ret = ManipulabilityMoveIt::initialize(config);
+
+  if (config.hasMember("exclusion_list") && config["exclusion_list"].getType() == XmlRpc::XmlRpcValue::TypeArray)
+  {
+    for (std::size_t i = 0; i < config["exclusion_list"].size(); ++i)
+    {
+      std::string excluded_link = static_cast<std::string>(config["exclusion_list"][i]);
+      exclusion_list_.push_back(excluded_link);
+    }
+
+    if (exclusion_list_.empty())
+    {
+      ROS_ERROR_STREAM("Exclusion list is empty");
+      return false;
+    }
```

Comment:
I think we should just remove this. It's probably okay if someone specifies and empty vector
```suggestion

```